javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -35,6 +35,11 @@ export interface IClient extends IClientCore {
   };
 }
 
+export interface ISearchParams{
```

Comment:
Is this about search params? or rather about listing/pagination?

Reply:
For now is for listing, but I assume that at some point we will need to filter by other parameters, for example, status for the voting proposals. We can change it to something like `IPaginationParams` or `IListingParams` if you prefer

Reply:
If "filter" sounds more reasonable to you, let's use it, but search is so specific... maybe we just want to list DAO's, with no filter at all